### PR TITLE
Adjust doc title

### PR DESCRIPTION
**Pull request by gastaldi**
This will remove the  `Quarkus ` prefix in the documentation title

### CODE REVIEW

File: docs/antora.yml [hunk 1]
```diff
@@ -1,5 +1,5 @@
 name: quarkus-micrometer-registry
-title: Quarkus Micrometer Registry extensions
+title: Micrometer Registry
```
Comment (brunobat) [inline docs/antora.yml:2]: I think the title is ok... This project is a multi module project with many Micrometer registries to use in Quarkus.

**Review by brunobat — CHANGES_REQUESTED** [review]

**Review by jonatan-ivanov — APPROVED** [review]

File: docs/antora.yml (hunk 1 above)
Comment (jonatan-ivanov) [reply docs/antora.yml]: I just wanted to ask if this repo isn't a set of Quarkus extensions for optional Micrometer registries?

Comment (brunobat) [reply docs/antora.yml]: Yes it is @jonatan-ivanov

**Review by brunobat — APPROVED** [review]
Thanks for the change @gastaldi

**Pull request merged**